docs/Running-Tachyon-on-EC2-Yarn.md

```diff
@@ -138,7 +138,7 @@ ApplicationMaster for Tachyon.
 ```bash
 $ cd /tachyon
 $ ./bin/tachyon-stop.sh
-$ mvn clean install -Dhadoop.version=2.4.1 -Pyarn -DskipTests
+$ mvn clean install -Dhadoop.version=2.4.1 -Pyarn -DskipTests -Dfindbugs.skip -Dmaven.javadoc.skip -Dcheckstyle.skip
```

Comment:
Only to speed up the compilation - the removed tasks are pointless when compiling on EC2, and most of the time gets spent on findbugs and javadoc generation.